.idea/modules/HackerRank.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add this guy to your .gitignore as we can easily re-create it using `./gradlew idea` when we check out.